ui/shared/components/panel/LoadWorkspaceDataForm.jsx

```diff
@@ -104,16 +104,37 @@ const createProjectFromWorkspace = postWorkspaceValues(
   window.location.href = `/project/${responseJson.projectGuid}/project_page`
 })
 
-const FORM_WIZARD_PAGES = [
-  { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') },
-  { fields: [DATA_BUCK_FIELD, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD], onPageSubmit: postWorkspaceValues('validate_vcf') },
+const addDataFromWorkspace = projectGuid => (values, onSuccess) => (
+  new HttpRequestHelper(`/api/project/${projectGuid}/add_workspace_data`, (responseJson) => {
+    if (responseJson.success) {
+      onSuccess()
+    }
+  }).post({ ...values, uploadedFileId: values.uploadedFile?.uploadedFileId }))
+
+const GRANT_ACCESS_PAGE = { fields: [AGREE_CHECKBOX], onPageSubmit: postWorkspaceValues('grant_access') }
+const VALIDATE_VCF_PAGE = {
+  fields: [DATA_BUCK_FIELD, SAMPLE_TYPE_FIELD, REQUIRED_GENOME_FIELD],
+  onPageSubmit: postWorkspaceValues('validate_vcf'),
+}
+
+const NEW_PROJECT_WIZARD_PAGES = [
+  GRANT_ACCESS_PAGE,
+  VALIDATE_VCF_PAGE,
   { fields: [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD] },
 ]
 
-const LoadWorkspaceDataForm = React.memo(({ params }) => (
+const ADD_DATA_WIZARD_PAGES = [
+  GRANT_ACCESS_PAGE,
+  { ...VALIDATE_VCF_PAGE, fields: [DATA_BUCK_FIELD] },
+  { fields: [UPLOAD_PEDIGREE_FIELD] },
+]
+
+const successMessage = 'Your request to load data has been submitted. Loading data from AnVIL to seqr is a slow process, and generally takes a week. You will receive an email letting you know once your new data is available.'
```

Comment:
I found the success messages for both the new Anvil project and new data are the same. So I added the message to the `LoadWorkspaceDataForm` so that it can be shared by both cases.

Reply:
there should never be a success message shown on the create page. It should redirect to the project page before anything can be shown

Reply:
okay